lib/driver/compilerTool.cpp

```diff
@@ -100,6 +102,7 @@ int Compiler([[maybe_unused]] int Argc,
            .add_option("enable-extended-const"sv, PropExtendConst)
            .add_option("enable-threads"sv, PropThreads)
            .add_option("enable-all"sv, PropAll)
+           .add_option("optimization-level"sv, OptimizationLevel)
```

Comment:
@hydai ,
Do you think we should use `-O0`, `-O1`, ...etc. like the `gcc` and `clang` compilers?

Reply:
In that case, we will have multiple options in the help text.
As in gcc, options on optimizations are not shown in the help text.

Reply:
We can use the `--optimize=[0,1,2,3,s,z]`. This is not for the common users, and we don't need to use the same option name as `gcc` and `clang` does.